doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -266,6 +285,37 @@ More on sorts can be found in Section :ref:`sorts`.
 Binders
 -------
 
+.. insertgram open_binders exclam_opt
+
+.. productionlist:: coq
+   open_binders : `name_list` : `term`
+                : `binder_list`
+   name_list : `name_list` `name`
+             : `name`
+   name : _
+        : `ident`
+   binder_list : `binder_list` `binder`
```

Comment:
`binder_list` used to be called `binders`. I don't mind the renaming but it should be substituted everywhere it is used in the manual.